Resources/Prototypes/Entities/Objects/Fun/Tabletop/nanopoliafigurines.yml

```diff
@@ -0,0 +1,913 @@
+- type: entity
+  id: NanapoliaTabletopPiece
+  parent: BaseTabletopPiece
+  abstract: true
+  components:
+  - type: Sprite
+    noRot: true
+    sprite: Objects/Fun/Tabletop/nanofigure.rsi
+
+- type: entity
+  id: FigurineCat
+  name: Cat
+  parent: NanapoliaTabletopPiece
+  components:
+  - type: Sprite
+    state: cat
+    
+- type: entity
+  parent: NanapoliaTabletopPiece
+  id: FigurineLaceups
+  name: Laceups
+  description: A figurine from nanopolia
+  components:
+  - type: Sprite
+    state: laceups
+    
+- type: entity
+  parent: NanapoliaTabletopPiece
+  id: FigurineMcgriff
+  name: Mcgriff
+  description: A figurine from nanopolia
+  components:
+  - type: Sprite
+    state: mcgriff
+
+- type: entity
+  parent: NanapoliaTabletopPiece
+  id: FigurineTopHat
+  name: Top Hat
+  description: A figurine from nanopolia
+  components:
+  - type: Sprite
+    state: top_Hat
+
+- type: entity
+  parent: NanapoliaTabletopPiece
+  id: FigurineGreenAutolathe
+  name: Figurine Green Autolat
```

Comment:
Green Autolathe Figurine, same for the rest of them

Reply:
Done